includes/class-wcpdf-settings-upgrade.php

```diff
@@ -0,0 +1,95 @@
+<?php
+namespace WPO\WC\PDF_Invoices;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+if ( !class_exists( '\\WPO\\WC\\PDF_Invoices\\Settings_Upgrade' ) ) :
+
+class Settings_Upgrade {
+
+	function __construct()	{
+		add_action( 'wpo_wcpdf_after_settings_page', array( $this, 'extension_overview' ), 10, 2 );
+	}
+
+	public function extension_overview( $tab, $section ) {
+		if ( $tab === 'upgrade' ) {
+			
+			$features = array(
+				array( 
+					'label' => __( 'Proforma invoice & Credit note', 'woocommerce-pdf-invoices-packing-slips' ),
+					'description' => __( 'Update your workflow and handle refunds.', 'woocommerce-pdf-invoices-packing-slips' ),
+					'extensions' => ['pro', 'bundle'],
+				),
+				array( 
+					'label' => __( 'Dropbox / FTP upload', 'woocommerce-pdf-invoices-packing-slips' ),
+					'description' => __( 'Automatically upload your documents.', 'woocommerce-pdf-invoices-packing-slips' ),
+					'extensions' => ['pro', 'bundle'],
+				),
+				array( 
+					'label' => __( 'Bulk export', 'woocommerce-pdf-invoices-packing-slips' ),
+					'description' => __( 'Easily export documents for a specific date range.', 'woocommerce-pdf-invoices-packing-slips' ),
+					'extensions' => ['pro', 'bundle'],
+				),
+				array( 
+					'label' => __( 'Multilingual support', 'woocommerce-pdf-invoices-packing-slips' ),
+					'description' => __( 'Handle document translations via either WPML or Polylang.', 'woocommerce-pdf-invoices-packing-slips' ),
```

Comment:
Maybe we could add this: "Note: It has limited compatibility with other multilanguage plugins."

Reply:
I think if we do not provide full compatibility we should not advertise it.